dali/operators/audio/nonsilence_op_test.cc

```diff
@@ -55,7 +55,9 @@ TEST_F(NonsilenceOpTest, DetectNonsilenceRegionTest) {
   auto nonsilence_region = detail::DetectNonsilenceRegion<float>(intermediate_buffer,
                                                                  {in, 0, 1.f, false,
                                                                   this->window_size_, -1});
-  ASSERT_EQ(nonsilence_region, nonsilence_region_);
+  // It's impossible to figure out where within the window the nonsilent region begins and ends
```

Comment:
here we should rather compare that the range is exactly where we expect it to be